pkg/scheduler/frameworkext/framework_extender.go

```diff
@@ -203,3 +218,19 @@ func PluginFactoryProxy(extendHandle ExtendedHandle, factoryFn frameworkruntime.
 		return factoryFn(args, extendHandle)
 	}
 }
+
+// gang test used
+type PodGroupClientSetAndHandle struct {
```

Comment:
Hey, If the type is just used to testing, please move the codes to the testing package.

Reply:
ok,done